persistence-typed/src/main/resources/reference.conf

```diff
@@ -47,6 +47,12 @@ pekko.persistence.typed {
   # this can be changed by setting this to 'true' in which case the internal logging is sent to
   # the actor context logger.
   use-context-logger-for-internal-logging = false
+
+  # Commands get stashed during persistence and un-stashed afterward, causing functions to be called
+  # recursively if commands are read-only (i.e. do not cause any persistence). This might cause a stack
+  # overflow in case there are many messages to unstash. Setting this to true enables a fix that turns
+  # recursion into a plain cycle, thus preventing stack overflows
+  break-recursive-calls-when-unstashing-read-only-commands = false
```

Comment:
can we call this `recurse-when-unstashing-read-only-commands` - opposite meaning to your flag (recurse=true is the old behaviour)? - I'd like it to default to false so that the new non-recursive code is the default